README.md

```diff
@@ -8,28 +8,17 @@ these members to elastic search.
 ## Installation
 
 
-## :nut_and_bolt: Installation && Development
-The `run.sh` script lets you different options to run various part of the program:
-```
-chmod +x ./run.sh
-./run.sh
-```
-Please note that if you want to run the integration test(s) you need an elastic search
-cluster running locally. In that case, before running any test do:
-```
-docker-compose up
-```
-Afterwards, you can run the integration test.
-> **Note**: To shut down the cluster: docker-compose down
+## :nut_and_bolt: Development
+To run the unit test(s) do `run-unit-tests.sh`
 
-If you want to format a file with _black_ tool, once in the docker's terminal
+To obtain a terminal with all the dependencies do `run-sh-with-pip-requirements.sh`.
+Then, you can do
 ```
 black path_to_your_file.py
-```
-Similar for pylint
-```
+# or
 pylint path_to_your_file.py
```

Comment:
I think we can remove pylint, we're leaning toward using **black** as the standard formatter for this repo

Reply:
These tools have different concerns:
- _pylint_ warns you about bad practice in your code
- _black_ format the code for you according to some standards

So, I would vote to keep them both.